src/grt/test/congestion2.guideok

```diff
@@ -8075,9 +8086,9 @@ req_msg[2]
 147000 176400 151200 193200 metal2
 147000 189000 151200 193200 metal2
 147000 189000 172200 193200 metal3
-168000 189000 172200 193200 metal3
-168000 189000 172200 193200 metal4
-168000 189000 172200 193200 metal5
+168000 189000 172200 193200 metal2
+168000 189000 172200 193200 metal1
+168000 189000 172200 193200 metal1
```

Comment:
It seems that we have a floating guide in this net.

Reply:
Yes, this was a small error on the code, the last commit should fix it